config.py

```diff
@@ -41,6 +44,17 @@ def set_overlay(new_overlays, directory=CURRENT_DIRECTORY):
         json.dump(configs, file, indent=2, sort_keys=True)
 
 
+def set_rotation(rotation, directory=CURRENT_DIRECTORY):
+    """Set rotation for device"""
+    configs = read_configs(directory)
+    if "device" in configs:
+        configs.pop("device")
```

Comment:
Does anyone know why this check is here? I saw it was in the set overlay but didn't really know why we're doing this.

Reply:
It's because `device` corresponds to an environment variable (`MHP_CAMERA`), which we don't want written into the JSON. It's super messy in fact missing some checks for newer environment variables. I reckon fine to leave for now but definitely something to look into tidying up